integration-testing/docker/cypress.Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM cypress/included:12.7.0
+FROM cypress/included:latest
```

Comment:
We have seen breaking changes with previous Cypress releases. They often require configuration or layout changes, especially across major versions. Are you against pinning to specific versions?

Reply:
Oh not at all. I'll pin to the latest working version.

Reply:
Thanks. That's essentially what we do elsewhere -- pin to a working version and then occasionally update after confirming that the new version doesn't break our use.